pkgs/misc/tmux-plugins/default.nix

```diff
@@ -648,6 +648,28 @@ in rec {
     inherit mkTmuxPlugin;
   };
 
+  t-smart-tmux-session-manager = mkTmuxPlugin {
+    pluginName = "t-smart-tmux-session-manager";
+    version = "unstable-2023-06-05";
```

Comment:
Can you use the tag of the latest version? Like https://github.com/NixOS/nixpkgs/blob/e59598bda7846853e96f479d900c0f4970b31346/pkgs/misc/tmux-plugins/default.nix#L460-L472

Reply:
i made some changes is this whay you meant!

Reply:
I think so. Thanks.

Do you have hangs with 2.8.0? On my computer it seems to hang when I use ctrl+b + T

Reply:
nope works fine for me!